javascript/javascript1/week3/note_taking_app.js

```diff
@@ -0,0 +1,62 @@
+// SAVE A NOTE
+const notepad = [];
+function saveNote(notes, content, id) {
+    const draft = {};
```

Comment:
There is a shorter way to do this:
```suggestion
    const draft = {
      content: content,
      id: id,
    };
```

Reply:
You're absolutely right. I rectfied this "way" of creating an object in the cactuos_io exercise when adding an activity:
`if ... {
...
} else {
        activities.push({
            date: new Date().toLocaleDateString("en-GB"),
            activity: activity,
            duration: duration
        })
    }`

The reason for which I chose that twisted code for the note taking app, is because I thought "how can JS know that in - for example - content: content, it has to keep the first word "content" as it is a property, and change only the second because it works as a parameter inside the saveNote function?". I still don't know how JS differentiates both but at least I know it doesn't mix up that information 🤣 But now that I'm thinking, what if I really wanted my function to change the name of the property? Should I create a new variable inside the function scope and assign it to an object property using dot notation?